lib/spacesuit/auth_middleware.ex

```diff
@@ -52,6 +52,8 @@ defmodule Spacesuit.AuthMiddleware do
     result =
         token
         |> Joken.token
+        # is it possible to verify against alternatives here?
+        # that would make our lives easier if we want to rotate the secret / switch the algorithm
```

Comment:
Currently it fetches the secret out of the application config. It would be easy to do anything else here we want.